test_runner/batch_others/test_auth.py

```diff
@@ -22,6 +22,25 @@ class AuthKeys:
     pub: bytes
     priv: bytes
 
+    def generate_management_token(self):
+        token = jwt.encode({"scope": "pageserverapi"}, self.priv, algorithm="RS256")
+
+        # jwt.encode can return 'bytes' or 'str', depending on Python version or type
+        # hinting or something (not sure what). If it returned 'bytes', convert it to 'str'
+        # explicitly.
+        if isinstance(token, bytes):
```

Comment:
nit: maybe split encoding and bytes/string check into its own function, and in this shortcut functions pass body of the token to this generic encode function